readthedocs/projects/models.py

```diff
@@ -1906,184 +1906,204 @@ def add_features(sender, **kwargs):
-            _('Clean all files used in the build process'),
+            _("Build: Clean all files used in the build process"),
         ),
         (
             UPDATE_CONDA_STARTUP,
-            _('Upgrade conda before creating the environment'),
+            _("Conda: Upgrade conda before creating the environment"),
         ),
         (
             CONDA_APPEND_CORE_REQUIREMENTS,
-            _('Append Read the Docs core requirements to environment.yml file'),
+            _("Conda: Append Read the Docs core requirements to environment.yml file"),
         ),
         (
             ALL_VERSIONS_IN_HTML_CONTEXT,
             _(
-                'Pass all versions (including private) into the html context '
-                'when building with Sphinx'
+                "Sphinx: Pass all versions (including private) into the html context "
+                "when building with Sphinx"
             ),
         ),
         (
             CACHED_ENVIRONMENT,
-            _('Cache the environment (virtualenv, conda, pip cache, repository) in storage'),
+            _(
+                "Build: Cache the environment (virtualenv, conda, pip cache, repository) in storage"
+            ),
         ),
         (
             LIMIT_CONCURRENT_BUILDS,
-            _('Limit the amount of concurrent builds'),
+            _("Build: Limit the amount of concurrent builds"),
         ),
         (
             CDN_ENABLED,
-            _('CDN support for a project\'s public versions when privacy levels are enabled.'),
+            _(
+                "Hosting: CDN support for a project's public versions when privacy levels "
+                "are enabled."
+            ),
         ),
         (
             DOCKER_GVISOR_RUNTIME,
-            _("Use Docker gVisor runtime to create build container."),
+            _("Build: Use Docker gVisor runtime to create build container."),
         ),
         (
             RECORD_404_PAGE_VIEWS,
-            _("Record 404s page views."),
+            _("Hosting: Record 404s page views."),
         ),
         (
             ALLOW_FORCED_REDIRECTS,
-            _("Allow forced redirects."),
+            _("Hosting: Allow forced redirects."),
         ),
         (
             DISABLE_PAGEVIEWS,
-            _("Disable all page views"),
+            _("Hosting: Disable all page views"),
         ),
         (
             DISABLE_SPHINX_DOMAINS,
-            _("Disable indexing of sphinx domains"),
+            _("Sphinx: Disable indexing of sphinx domains"),
         ),
         (
             RESOLVE_PROJECT_FROM_HEADER,
-            _("Allow usage of the X-RTD-Slug header"),
+            _("Hosting: Allow usage of the X-RTD-Slug header"),
         ),
         (
             USE_UNRESOLVER_WITH_PROXITO,
-            _("Use new unresolver implementation for serving documentation files."),
+            _(
+                "Hosting: Use new unresolver implementation for serving documentation files."
```

Comment:
Maybe proxito instead of hosting for all these?

Reply:
Done!